corehq/apps/api/resources/meta.py

```diff
@@ -34,16 +62,13 @@ def accessed(self, identifier, **kwargs):
         # only required when using this throttling mechanism.
         from tastypie.models import ApiAccess
 
-        # only record in redis if we need the throttle, otherwise skip
-        # and just leave the db logging
-        if not API_THROTTLE_WHITELIST.enabled(identifier):
-            super(CacheDBThrottle, self).accessed(identifier, **kwargs)
+        api_rate_limiter.report_usage(identifier.domain)
         # Write out the access to the DB for logging purposes.
         url = kwargs.get('url', '')
         if len(url) > 255:
             url = url[:251] + '...'
         ApiAccess.objects.create(
-            identifier=identifier,
+            identifier=identifier.username,
             url=url,
             request_method=kwargs.get('request_method', '')
         )
```

Comment:
side note: I don't think we've ever used this data so I'd be pro removing it

Reply:
Is this basically a log object notes some data about each API request? If it contains anything about the request that we don't already have in auditcare, it seems like a useful auditing record. It looks like all these fields are also captured in auditcare though, thus this doesn't add any additional value. Is that basically the logic, @snopoke? If so then I agree, just want to make sure I'm following correctly.

Reply:
@dannyroberts that's correct, this doesn't capture anything more than what's in auditcare

Reply:
We do also plan on adding individual request logging as part of the [generic inbound API work](https://docs.google.com/document/d/1lpGyVzur2G2lFOQJJ2nNiNQzasZFMccJiFtRxS5CSPo/edit#heading=h.7jj84een0rk8).  That'd need to store more than this does, though.